.github/workflows/docker-image.yml

```
@@ -39,9 +39,17 @@
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Set APP_VERSION env
+        run: echo APP_VERSION=$(echo ${GITHUB_REF} | rev | cut -d'/' -f 1 | rev ) >> ${GITHUB_ENV}
+      - name: Set APP_RELEASE env
+        run: echo APP_RELEASE=$(( ($(date +%s) - $(date -d "2024-12-07" +%s)) / 86400 )) >> ${GITHUB_ENV}
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:80: Double quote to prevent globbing and word splitting

(shellcheck)